blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/template/views/filesystem/FileSystemViewFactory.java

```diff
@@ -12,19 +14,18 @@
     private FileSystemViewFactory() {
     }
 
-    public static FileSystemView convertToFileSystem(FileSystemConfigurationView fileSystemConfigurationView) {
+    public static FileSystemView convertToFileSystem(@Nonnull FileSystemConfigurationView fileSystemConfigurationView) {
         FileSystemConfiguration fileSystemConfiguration = fileSystemConfigurationView.getFileSystemConfiguration();
-        if (fileSystemConfiguration != null) {
-            if (fileSystemConfiguration instanceof GcsFileSystemConfiguration) {
-                return new GcsFileSystemView(fileSystemConfigurationView);
-            } else if (fileSystemConfiguration instanceof WasbFileSystemConfiguration) {
-                return new WasbFileSystemView(fileSystemConfigurationView);
-            } else if (fileSystemConfiguration instanceof AdlsFileSystemConfiguration) {
-                return new AdlsFileSystemView(fileSystemConfigurationView);
-            }
+        if (fileSystemConfiguration instanceof GcsFileSystemConfiguration) {
```

Comment:
Would be nicer to create 3 different Converter, but would be more better to use MapSruct .